api/audit/events_api.go

```diff
@@ -28,7 +28,7 @@ func NewEventService(concertoService utils.ConcertoService) (*EventService, erro
 func (cl *EventService) GetEventList() (events []types.Event, err error) {
 	log.Debug("GetEventList")
 
-	data, status, err := cl.concertoService.Get("/v1/audit/events")
+	data, status, err := cl.concertoService.Get("/v2/audit/events")
```

Comment:
Sure, i was working on it for a time. Finally i decided not to include in current issue. (but my idea is to unify it in close future to avoid this)
I have an eye on it.